tap_mysql/sync_strategies/common.py

```diff
@@ -109,8 +106,9 @@ def to_utc_datetime_str(val):
         # The mysql-replication library creates naive date and datetime objects
         # which will use the local timezone thus we must set tzinfo accordingly
         # See: https://github.com/noplay/python-mysql-replication/blob/master/pymysqlreplication/row_event.py#L143-L145
-        timezone = tzlocal.get_localzone()
-        the_datetime = timezone.localize(the_datetime)
+
+        #NB> this code will only work correctly when the local time is set to UTC because of the method timestamp()
+        the_datetime = datetime.datetime.fromtimestamp(the_datetime.timestamp(), pytz.timezone('UTC'))
```

Comment:
Isn't this the same as `the_datetime.replace(tzinfo=pytz.UTC)` if local time is set to UTC and it's a naive datetime that's assumed UTC by the driver?